### PR TITLE
docs: update supported versions in SECURITY.md

### PR DESCRIPTION
**Pull request by kwvg**
## Additional Information

Updates the supported versions list in `SECURITY.md`

## Checklist:

- [x] I have performed a self-review of my own code **(note: N/A)**
- [x] I have commented my code, particularly in hard-to-understand areas **(note: N/A)**
- [x] I have added or updated relevant unit/integration/functional/e2e tests **(note: N/A)**
- [x] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 | Version | Supported          |
 | ------- | ------------------ |
-| 21      | :white_check_mark: |
-| 20.1    | :white_check_mark: |
-| < 20.1  | :x:                |
+| 22      | :white_check_mark: |
+| 21.1    | :white_check_mark: |
```
Comment (knst) [inline SECURITY.md:8]: I think 21.1 should not be supported because we have fork `withdrawal` which is not implemented for 21.1

Comment (kwvg) [reply SECURITY.md]: Resolved in latest push

**Review by knst — APPROVED** [review]
utACK 87b3c7f5d19291c80c078a3a63ceb2f22cc3b666

**Review by UdjinM6 — APPROVED** [review]
utACK 87b3c7f5d19291c80c078a3a63ceb2f22cc3b666

**Pull request merged**